draft-erdtman-cleartext-jwe.xml

```diff
@@ -214,7 +214,7 @@
         the root and at the recipients level, then the recipients level has precedence.
 	<!-- TBD Rather than defining precedence, why not prohibit duplicate members across levels? -->
 
-        <figure align="center" anchor="fig:KeyEncryptionExamplewithonerecipient" title='Key Encryption Example with one recipient'>
+        <figure align="center" anchor="fig:KeyEncryptionExamplewithonerecipient" title='Key Encryption Example with two recipients'>
```

Comment:
Title casing rules say that this should actually say "Key Encryption Example with Two Recipients".

Reply:
Thanx Mike, it sounds quite logical as well :-)  I'll fix it